### DISCUSSION TITLE
Http Proxy Manager Logs Sensitive Information

### DISCUSSION THREAD

**Discussion by zachhutchins** [mia-platform/community#515] [Ideas]
### Problem

Http Proxy Manager when debug logs are enabled will log just about the entire request, including credentials. All credentials' data should be asterisked out by default. For example, if you are using Basic Authorization the application will log a username and password pair in plain text. I understand in the documentation it calls out to not use in production, however I feel this is an easy thing to make a mistake on and let credentials slip into logs as plain text.

---

### Solution

Authorization Header and all other places that you can reasonably expect credentials to be is asterisked out.

**Comment by fredmaggiowski** [comment]
I think the information was originally included in the debug log for sake of helping debugging possible issues with the authentication credentials. I personally agree with you tbh, this has been tracked in the internal backlog
